### PR TITLE
Add sharpen.py and related tests

### PR DESCRIPTION
**Pull request by franklu2014**
Add sharpen.py and a related jupyter noteboob for initial test.  I will add docstring when the code settles down.

### CODE REVIEW

File: group_12/sharpen.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      the image to be processed
+
+    Returns
+    --------
+    img : a (H,W) numpy array, returns the sharpened and monotoned image
+
+    """
+    img2 = rgb2gray(img)
+    flt = -np.ones((3,3))/22
+    flt[1,1] = 1
```
Comment (rita-ni) [inline group_12/sharpen.py]: May add some short comments to explain what is the transforming function doing. I would be better if some error handling tests can be added to the function to check input values/types.

File: tests/test_sharpen.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from group_12 import sharpen
+import matplotlib.pyplot as plt
+
```
Comment (rita-ni) [inline tests/test_sharpen.py:5]: You may want to add some tests corresponding to error handling tests in the function to check if your function handles invalid inputs correctly. Some short comments explaining what each test function is doing would also be great! The other tests look good.

**Review by rita-ni — CHANGES_REQUESTED** [review]

**Review by rita-ni — APPROVED** [review]

File: group_12/sharpen.py (hunk 1 above)
Comment (franklu2014) [reply group_12/sharpen.py]: That's a perfect suggestion.  I will do that.

File: tests/test_sharpen.py (hunk 2 above)
Comment (franklu2014) [reply tests/test_sharpen.py:5]: You are right. I will add few more tests for capturing errors.

**Pull request merged**